### ISSUE TITLE
Preview Mode: Doesn't show errors

### ISSUE THREAD

**Issue by kuruk-mm** [decentraland/unity-renderer#705]
When we have a fatal error in PREVIEW, we don't show anything to the user.

![image.png](https://images.zenhubusercontent.com/60647b38cc4b79842895cb82/5b2b0868-74ef-4232-9fae-40966668e380)

How to reproduce:
- create empty scene `dcl init` -> `dcl start`
- open tab with this URL: `http://127.0.0.1:8000/?ws=asdasd` (`ws=asdasd` will produce a fatal error)

we should see this screen:
![image.png](https://images.zenhubusercontent.com/60647b38cc4b79842895cb82/0c446dfe-045a-4884-9307-5216b3f5c079)

**Comment by JorgeDecentraland** [comment]
@menduz should we transfer this ticket to the sdk board?

**Comment by menduz** [comment]
Yes please, notify the team in the channel so it doesn't get lost
